admin/import/class-import-settings.php

```diff
@@ -105,7 +105,12 @@ private function determine_path() {
 		$this->path = $this->upload_dir['basedir'] . DIRECTORY_SEPARATOR . 'wpseo-import' . DIRECTORY_SEPARATOR;
 
 		if ( ! isset( $GLOBALS['wp_filesystem'] ) || ! is_object( $GLOBALS['wp_filesystem'] ) ) {
-			WP_Filesystem();
+			$url = wp_nonce_url(
+				self_admin_url('admin.php?page=wpseo_tools&tool=import-export'),
```

Comment:
the build fails for unmet coding standards: missing spaces between brackets

Reply:
9f67746